whisper/whisperv5/api.go

```diff
@@ -562,7 +562,7 @@ func (api *PublicWhisperAPI) NewMessageFilter(req Criteria) (string, error) {
 	}
 
 	if len(req.Topics) > 0 {
-		topics = make([][]byte, 1)
+		topics = make([][]byte, 0, 1)
```

Comment:
it makes sense to create topics slice with the same capacity as req.Topics length, isn't it?

Reply:
done